src/core/etl/src/Flow/ETL/Row/Entry/JsonEntry.php

```diff
@@ -6,51 +6,39 @@
 
 use Flow\ArrayComparison\ArrayComparison;
 use Flow\ETL\Exception\InvalidArgumentException;
+use Flow\ETL\PHP\Type\Native\ArrayType;
+use Flow\ETL\PHP\Type\Type;
 use Flow\ETL\Row\Entry;
 use Flow\ETL\Row\Reference;
 use Flow\ETL\Row\Schema\Definition;
 
 /**
- * @implements Entry<string, array{name: string, value: array<mixed>, object: boolean}>
+ * @implements Entry<string, array{name: string, value: array, object: boolean}>
  */
 final class JsonEntry implements \Stringable, Entry
 {
     use EntryRef;
 
-    private bool $object;
+    private array $value;
 
     /**
-     * JsonEntry constructor.
-     *
-     * @param array<mixed> $value
-     *
      * @throws InvalidArgumentException
      */
-    public function __construct(private readonly string $name, private readonly array $value)
+    public function __construct(private readonly string $name, string|array $value, private bool $object = false)
```

Comment:
what's the point of passing `$object` through constructor, when it's overwritten anyway?